### ISSUE TITLE
Immediate Hydration on Page Load and Toast feedback on long running USQL queries 

### ISSUE THREAD

**Issue by mcrascal** [evidence-dev/evidence#1648]
## Feature Description

Today, universal SQL waits until you interact with an input to execute the initial SQL on the page. If the page has large data sources, the response time on the first interaction is guaranteed to be much longer than necessary. The difference in response speed for the first interaction and the second interaction is especially large if the page is taking advantage of predicate pushdown where the second interaction stays within the chunk of parquet that was already delivered. 

It would be preferable to execute these queries immediately to decrease the chance that a user hits a slow interaction. 

Where a long running query is running, we should flag that to the layout so that it can display a loading toast, or similar feedback.

**Comment by ItsMeBrianD** [comment]
We can add a `.backgroundFetch` function to the QueryStore and use it as such:

```javascript
onMount(async () => {
    await tick() // wait for all html updates to finish
    await new Promise((r) => setTimeout(r, 0)) // get to the end of the event loop
    // each query...
    await query.backgroundFetch() // Runs the query but ignores the results
    //
})

```

**Comment by csjh** [comment]
Closed by #1699
